specification/app/resource-manager/Microsoft.App/preview/2022-11-01-preview/ContainerAppsJobs.json

```diff
@@ -0,0 +1,861 @@
+    },
+    "ContainerAppsJobPatchProperties": {
+      "type": "object",
+      "description": "ContainerApp resource specific properties.",
+      "properties": {
+        "identity": {
+          "description": "Managed identities needed by a container app job to interact with other Azure services to not maintain any secrets or credentials in code.",
+          "$ref": "../../../../../common-types/resource-management/v3/managedidentity.json#/definitions/ManagedServiceIdentity"
+        },
+        "properties": {
+          "type": "object",
+          "properties": {
+            "provisioningState": {
+              "description": "Provisioning state of the Container Apps Job.",
+              "enum": [
+                "InProgress",
+                "Succeeded",
+                "Failed",
+                "Canceled",
+                "Deleting"
+              ],
+              "type": "string",
+              "readOnly": true,
+              "x-ms-enum": {
+                "name": "ContainerAppsJobProvisioningState",
+                "modelAsString": true
+              }
+            },
+            "environmentId": {
+              "description": "Resource ID of environment.",
+              "type": "string",
+              "x-ms-mutability": [
+                "create",
+                "read"
+              ]
+            },
+            "configuration": {
+              "$ref": "#/definitions/JobConfiguration",
+              "description": "Container Apps Job configuration properties."
+            },
+            "template": {
+              "$ref": "./CommonDefinitions.json#/definitions/ContainerAppsJobTemplate",
+              "description": "Container Apps job definition."
+            },
+            "outboundIpAddresses": {
+              "description": "Outbound IP Addresses of a container apps job.",
+              "type": "array",
+              "items": {
+                "type": "string"
+              },
+              "readOnly": true
+            },
+            "eventStreamEndpoint": {
+              "description": "The endpoint of the eventstream of the container apps job.",
+              "type": "string",
+              "readOnly": true
+            }
+          }
+        }
+      }
+    },
+    "ContainerAppsJobsCollection": {
+      "description": "Container Apps Jobs collection ARM resource.",
+      "required": [
+        "value"
+      ],
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Collection of resources.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ContainerAppsJob"
+          }
+        },
+        "nextLink": {
+          "description": "Link to next page of resources.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "Dapr": {
+      "description": "Container Apps Job Dapr configuration.",
+      "type": "object",
+      "properties": {
+        "enabled": {
```

Comment:
>enabled [](http://example.com/codeflow?start=9&length=7)

Recommend enums (in general) over booleans, e.g. activeState: { Active, Disabled } or similar, or even enabled: { True, False } if you prefer. Ref: [https://armwiki.azurewebsites.net/rp_onboarding/process/api_review_best_practices.html?q=boolean#common-issues-found-in-review](https://armwiki.azurewebsites.net/rp_onboarding/process/api_review_best_practices.html?q=boolean#common-issues-found-in-review) for more details.